CBLClient/Apps/CBLTestServer-Java-Desktop/build.gradle

```diff
@@ -0,0 +1,66 @@
+plugins {
+    id 'java'
+}
+
+group 'com.couchbase.mobiletestkit'
+
+ext {
+    DEF_VERSION = "2.7.0-56"
+    COUCHBASE_LITE_JAVA_VERSION = System.getProperty('version', DEF_VERSION)
+
+    PROJECT_DIR = "${projectDir}"
```

Comment:
Most of our build files have the ability to be built from multiple locations.  This is essential if this code will be built as part of the Jenkins build for the Java product.  In order to do that, they let the root build file define ROOT_DIR and PROJECT_DIR.

Reply:
done some clean up, removed ROOT_DIR and PROJECT_DIR in my code